spec/fixtures/locations.json

```diff
@@ -14,6 +14,7 @@
       "properties": {
         "title": "Belfast Citizens Advice Bureau",
         "address": "Merrion Business Centre, 58 Howard St, Belfast, BT1 6PJ",
+        "booking_centre": "928543c7-e8f0-45b2-a300-c2f996b007d9",
```

Comment:
It would be a good thing to do though requires a fair bit of rejigging of the JS app.

I've opened [locator#20](https://github.com/guidance-guarantee-programme/locator/issues/20) to track this.

Stick with `booking_centre` until this is done?